packages/graphql/src/translate/where/property-operations/create-connection-operation.ts

```diff
@@ -197,6 +197,7 @@ export function createConnectionWherePropertyOperation({
     Object.entries(whereInput).forEach(([key, value]) => {
         if (isLogicalOperator(key)) {
             const subOperations: (Cypher.Predicate | undefined)[] = [];
+            value = Array.isArray(value) ? value : [value];
```

Comment:
Consider using `asArray` here, it should also fix the typings so there is no need to cast `asArray<any>`